options/ansi/include/memory.h

```diff
@@ -0,0 +1,9 @@
+
+#ifndef _MEMORY_H
+#define _MEMORY_H
+
+#ifndef _STRING_H
```

Comment:
Why this? ``string.h`` already has include guards.

Reply:
It matches with what glibc does, so I copied that.

Reply:
It's not necessary though; let's remove it.